content/guides/component-testing/mounting-vue.md

```diff
@@ -80,6 +80,21 @@ describe('<Stepper>', () => {
 
 Here, we have a single test that ensures that our component mounts.
 
+If you already have a component that you want to create a test for and you don't
+have a spec pattern configured for your project, we can scaffold the test for
```

Comment:
>and you don't have a spec pattern configured for your project

The restriction around needing to "not have a spec pattern" will only exist until next week, wonder if we should hold off on the docs until then? If not, I'd suggest this should be more of a warning format up top, it's a bit easy to skim over here, but makes the difference between this feature being present or not.

> we can scaffold the test for you via the Cypress App

Should we explain where this will be scaffolded? 

One minor thing to confirm: I think we refer to the files as _specs_ in the docs and only use "test" to talk about actual tests contained in the spec file (`it('whatever')`). That's probably the right wording to use here - "create a spec for", "scaffold the spec". The precision is probably more important for dashboard customers, where what we mean by "test" is a billing thing.

Reply:
correct, we refer to the collection of tests as a spec.

Reply:
To clarify:

- Spec: a file on your computer
- Suite: `describe` (can contain 0 or more suites and tests)
- Test: `it`

This is the common definitions across all test runners - we should definitely make sure we use the right definitions.

Reply:
Okay I updated it to say "spec" instead of "test", and I removed the 

> and you don't have a spec pattern configured for your project

condition, since this will be removed in the next release. I'll target `develop` so that this doc will go out with the next release